### PR TITLE
SAK-31944 Fix Calendar tool Month display name

### PR DESCRIPTION
**Pull request by master-bob**
See #3582 for previous discussion.

### CODE REVIEW

File: calendar/calendar-util/util/src/java/org/sakaiproject/util/CalendarUtil.java [hunk 1]
```diff
@@ -23,6 +23,7 @@
 
 import java.text.DateFormat;
 import java.text.SimpleDateFormat;
+import java.time.*;
```
Comment (steveswinsburg) [inline calendar/calendar-util/util/src/java/org/sakaiproject/util/CalendarUtil.java]: Can you expand this out please?

Comment (master-bob) [inline calendar/calendar-util/util/src/java/org/sakaiproject/util/CalendarUtil.java]: Sure, but it'll have to be on #3582 (which I'll have to re-open).

Comment (master-bob) [inline calendar/calendar-util/util/src/java/org/sakaiproject/util/CalendarUtil.java]: Done. Fixed in #3582.

**Comment by master-bob** [comment]
Closing this as I can't push to the branch.

**Pull request closed without merging**